examples/dreambooth/train_dreambooth.py

```diff
@@ -676,8 +676,8 @@ def collate_fn(examples):
                     if accelerator.is_main_process:
                         pipeline = DiffusionPipeline.from_pretrained(
                             args.pretrained_model_name_or_path,
-                            unet=accelerator.unwrap_model(unet),
-                            text_encoder=accelerator.unwrap_model(text_encoder),
+                            unet=accelerator.unwrap_model(unet, True),
+                            text_encoder=accelerator.unwrap_model(text_encoder, True),
```

Comment:
Great catch! 

The `keep_fp32_wrapper` argument was added only recently to `accelerate.unwrap_model` , so here we should first check if the method accepts that argument to keep compatibility with previous versions of accelerate.

It could be done with 

```
accepts_keep_fp32_wrapper = "keep_fp32_wrapper" in set(inspect.signature(accelerator.unwrap_model().parameters.keys())
```